internal/page/lpa_progress_tracker.go

```diff
@@ -45,16 +64,16 @@ func (pt ProgressTracker) Progress(lpa *lpastore.Lpa) Progress {
 			"certificateProviderSigned": pt.Localizer.T("theCertificateProviderHasDeclared"),
 			"attorneysSigned":           pt.Localizer.T("allAttorneysHaveSignedTheLpa"),
 			"lpaSubmitted":              pt.Localizer.T("opgHasReceivedTheLPA"),
+			"noticesOfIntentSent":       "weSentAnEmailTheLpaIsReadyToRegister",
 			"statutoryWaitingPeriod":    pt.Localizer.T("theWaitingPeriodHasStarted"),
 			"lpaRegistered":             pt.Localizer.T("theLpaHasBeenRegistered"),
 		}
 	} else {
 		labels = map[string]string{
-			"paid":                   "",
-			"confirmedID":            "",
 			"donorSigned":            pt.Localizer.T("youveSignedYourLpa"),
 			"attorneysSigned":        pt.Localizer.Count("attorneysHaveDeclared", len(lpa.Attorneys.Attorneys)),
 			"lpaSubmitted":           pt.Localizer.T("weHaveReceivedYourLpa"),
+			"noticesOfIntentSent":    "weSentAnEmailYourLpaIsReadyToRegister",
```

Comment:
Did we come to a decision yet on status changes vs notification sent for showing this?

Reply:
Sirius will change the status in lpa-store, that will then cause an event for us? (Is that what you mean?)